### PR TITLE
remove glob dependency

### PR DESCRIPTION
**Pull request by Uzlopak**
we dont need glob ;)

### CODE REVIEW

**Review by gr2m — COMMENTED** [review]
I'm okay either way, I think the code was easier to grasp using `glob`. Is there a particular reason why you would want to remove it? It's just a dev dependency anyway

File: scripts/build.mjs [hunk 1]
```diff
@@ -24,15 +36,6 @@ async function main() {
...
 
-  // Remove the types file from the dist-src folder
-  const typeFiles = await glob([
-    "./pkg/dist-src/**/types.js.map",
-    "./pkg/dist-src/**/types.js",
-  ]);
-  for (const typeFile of typeFiles) {
-    await rm(typeFile);
-  }
-
```
Comment (wolfy1339) [inline scripts/build.mjs:35]: Why was this removed? Whenever we have a `types.ts` file it would create an empty `types.js` file which is useless

**Comment by Uzlopak** [comment]
There is no types.ts file.
